app/(main)/generate/TabStep.tsx

```diff
@@ -15,9 +15,10 @@ export default function TabStep({ stepNumber, hidden, text }: Props) {
     <TabsTrigger
       value={`step-${stepNumber}`}
       className={`
-       grid
+      grid
```

Comment:
make sure you minimize changes. Only commit necessary changes. Unless you're changing the whitespace for every tailwind class, you should leave this one out.